.github/workflows/docker.yml

```diff
@@ -0,0 +1,32 @@
+name: "docker"
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  bake:
+    name: "build and pushing convco docker image"
+    runs-on: ubuntu-18.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
I prefer to set tags to a specific version rather than the latest. it's generally safer as you know exactly which packages are available by default in the distro. we can use `20.04` which is what `latest` is pointing to now